docs/source/nextflow-workflow.rst

```diff
@@ -135,6 +135,143 @@ Example ``sample_sheet.csv``
+
+Menu 4:
+Now we need to give Autometa the full path to where to find our sample sheet, where to put the output (it will generate a folder per metagenome in this folder) and where to store all the log files (aka the trace files)
+
+.. image:: ../img/Menu4.png
+
+Menu 5:
+This is where you can change your binning parameters. If you're not sure what you're doing I would recommend only changing the "length_cutoff". 
+The default cutoff is 3000bp, which means that any contigs/scaffolds smaller than 3000bp will not be considered for binning. This cutoff will depend on how good your assembly is: E.g. if your N50 is 1200 bp, I would chose a cutoff of 1000. If your N50 is more along the lines of 5000, I would leave the cutoff at the default 3000. I would strongly recommend against choosing a number below 900 here. In the example below, I have chosen a cutoff of 1000bp as my assembly wasn't particularly great
+
+.. image:: ../img/Menu5.png
+
+Menu 6:
+Here you have a choice to make:
+By enabling taxonomy aware mode, Autometa wll attempt to use taxonomic data to make your bins more accurate. However, this is a more computational expensive step and will make the process take longer. By leaving this option as the default "False" option, Autometa will bin according to kmer patterns.
+
+With either choice, you need to provide a path to where the `necessary databases <https://autometa.readthedocs.io/en/latest/databases.html>`_ are stored in the "single_db_dir" option. Here, I have enabled the taxonomy aware mode and provided the path to where the databases are stored:
+
+.. image:: ../img/Menu6.png
+
+Menu 7:
+This will depend on the computational resources you have available. You could start with the default values and see how the binning goes. If you have particularly complex datasets you may want to bump this up a bit. For your average metagenome (i.e. NOT Tb worth of assembled data - you need `large data mode for that <https://autometa.readthedocs.io/en/latest/scripts/binning/large_data_mode.html?highlight=large%20data%20mode>`_), you won't need more than 150Gb of memory. I've opted to use 75 Gb as a starting point for a few biocrust (somewhat diverse) metagenomes. 
+
+.. image:: ../img/Menu7.png
+
+Last few options:
+You will now be presented with a choice. If you are NOT using a scheduler, you can go ahead an type y to launch the workflow. If you are using a scheduler, type n - we have one more step to go. In the example below, I am using a scheduler so I have typed n to deny the immediate run.
+
+.. image:: ../img/launch_choice.png
+
+If you recall, we created a file called :code:`slurm_nextflow.config` that contains the information Autometa will need to talk to the Slurm scheduler. We need to include that file using the :code:`-c` flag (or configuration flag). Therefore to launch the Autometa run I would run the following command (you would change the :code:`/home/sam/Useful_scripts/` file path to what is appropriate for your system):
```

Comment:
Can you break this up into multiple lines to avoid the text-wrapping?

```suggestion
If you recall, we created a file called :code:`slurm_nextflow.config` that contains the information Autometa will need to talk to the Slurm scheduler. We need to include that file using the :code:`-c` flag (or configuration flag). Therefore to launch the Autometa workflow I would run the following command (you would change the :code:`/home/sam/slurm_nextflow.config` file path to what is appropriate for your system):
```